### PR TITLE
feat: Bump release version length to 250

### LINKED ISSUE #79: How can I integrate the codepush CLI with Sentry, to push new sourcemaps?
Hi,

I've seen this documentation: https://docs.sentry.io/clients/react-native/codepush/

I've set up this code in my app, but I'm confused about what I should be doing after I deploy a new release with CodePush. How can I get CodePush to send the new sourcemaps to Sentry?

Is there a Sentry integration for the codepush-cli, or a separate script I can run?

### PR DESCRIPTION
**Pull request by mitsuhiko**
This fixes https://github.com/getsentry/react-native-sentry/issues/79

### CODE REVIEW

**Review by mattrobenolt — CHANGES_REQUESTED** [review]
So we can’t run this migration in production without locking the whole table to rewrite the column.

What we can do, we’ve done elsewhere. For ourselves, we migrate the column to `text` which I believe can be done without rewriting the whole table. Then you just put a constraint on that column for the length. Basically exactly what is happening now, but in a safer manner.

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR includes migrations</td>
    </tr>
  </tbody>
</table>

## Migration Checklist

- [ ] new columns need to be nullable (unless table is new)
- [ ] migration with any new index needs to be done concurrently
- [ ] data migrations should not be done inside a transaction
- [ ] before merging, check to make sure there aren't conflicting migration ids


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by mitsuhiko** [comment]
@mattrobenolt yep. text does not need that. for some reason i thought that also applies to varying changes. Do you know if we have a migration where we did this already?

**Comment by mitsuhiko** [comment]
Turns out this is fine after 9.2

File: src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py [hunk 1]
```diff
@@ -0,0 +1,980 @@
...
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py]: As discussed, flip this back to True and I'll just run the SQL by hand to make sure so I can respond quicker if there were an issue.

File: src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py [hunk 2]
```diff
@@ -0,0 +1,980 @@
...
+
+    def forwards(self, orm):
+        db.alter_column('sentry_release', 'version', self.gf(
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py]: Looks like you need to regenerate this migration anyways to pick up the `ref` column change.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Comment by mitsuhiko** [comment]
@mattrobenolt turned on

File: src/sentry/models/release.py [hunk 3]
```diff
@@ -60,7 +61,7 @@ class Release(Model):
     )
     # DEPRECATED
     project_id = BoundedPositiveIntegerField(null=True)
-    version = models.CharField(max_length=64)
+    version = models.CharField(max_length=VERSION_LENGTH)
     # ref might be the branch name being released
     ref = models.CharField(max_length=64, null=True, blank=True)
```
Comment (mattrobenolt) [inline src/sentry/models/release.py]: Aren't you altering this field too?

Comment (mitsuhiko) [reply src/sentry/models/release.py]: @mattrobenolt decided against it for now.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**